src/CommunityToolkit.Maui.Core/Essentials/Badge/BadgeImplementation.windows.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace CommunityToolkit.Maui.ApplicationModel;
+
+using Windows.Data.Xml.Dom;
+using Windows.UI.Notifications;
+
+/// <inheritdoc />
+public class BadgeImplementation : IBadge
+{
+	/// <inheritdoc />
+	public void SetCount(int count)
+	{
+		var badgeUpdater = BadgeUpdateManager.CreateBadgeUpdaterForApplication();
+		if (count <= 0)
+		{
+			badgeUpdater.Clear();
+		}
+		else
+		{
+			var badgeXml = BadgeUpdateManager.GetTemplateContent(BadgeTemplateType.BadgeNumber);
+
+			var badgeElement = badgeXml.SelectSingleNode("/badge") as XmlElement;
+			badgeElement?.SetAttribute("value", count.ToString());
+
+			var badge = new BadgeNotification(badgeXml);
+			badgeUpdater.Update(badge);
+		}
+	}
+
+	/// <inheritdoc />
+	public int GetCount()
+	{
+		var badgeUpdater = BadgeUpdateManager.CreateBadgeUpdaterForApplication();
+		return 0;
```

Comment:
Windows' `GetCount()` method always returns `0` instead of the actual badge count

Reply:
it is not supported. removed GetCount method